CMakeLists.txt

```diff
@@ -315,7 +315,7 @@ endif()
 # =============================================================================
 if(DOXYGEN_FOUND AND SPHINX_FOUND)
   add_custom_target(
-          docs
+          corenrn_docs
```

Comment:
Need to update this in `.travis.yml`, `docs/README.md` and the following line in `CMakeLists.txt`: 
```
message(STATUS "make docs           | Build full docs. Calls targets: doxygen, sphinx")
```

Reply:
Thanks.